packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -49,6 +54,21 @@ const addCreatedByRolesPopulate = (populate) => {
   };
 };
 
+/**
+ * When webhooks.populateRelations is set to true, populated relations
+ * will be passed to any webhook event. The entity-manager
+ * response will not have the populated relations though.
+ * For performance reasons, it is recommended to set it to false,
+ *
+ * TODO V5: Set to false by default.
+ * TODO V5: Make webhooks always send the same entity data.
+ */
+const isRelationsPopulateEnabled = () => {
+  return strapi.config.get('server.webhooks.populateRelations', true);
+};
+
+const getCountDeepPopulate = (uid) => getDeepPopulate(uid, { countMany: true, countOne: true });
```

Comment:
Do you think this function is necessary? It feels to me that it obfuscates what is happening (meaning counting both for `countMany` and `countOne`)